src/main/java/io/jenkins/plugins/luxair/ImageTagParameterDefinition.java

```diff
@@ -38,24 +38,26 @@
     private final String credentialId;
     private String defaultTag;
     private Ordering tagOrder;
+    private Boolean verifySsl;
     private String errorMsg = "";
 
     @DataBoundConstructor
     @SuppressWarnings("unused")
     public ImageTagParameterDefinition(String name, String description, String image, String filter,
-                                       String registry, String credentialId) {
-        this(name, description, image, filter, "", registry, credentialId, config.getDefaultTagOrdering());
+                                       String registry, String credentialId, Boolean verifySsl) {
```

Comment:
If you add the parameter to the  `@DataBoundConstructor` you alter how this parameter gets created and therefor change/break API.
This would have breaking implications to how the `JobDsl` would handle this param then (all of the sudden **requiring** the verifySSL flag), since you already added the `@DataBoundSetter` to the property i would remove this from the `@DataBoundConstructor` (this makes it a optional parameter in the `JobDsl`).

I triggered the same implications once and had to update a lot of seed jobs ;)

Reply:
Tested localled and have committed your suggestion